testdata/sqllogictests/functions/current_schemas.slt

```diff
@@ -16,15 +16,17 @@ create schema current_schemas_test;
 statement ok
 set search_path = 'public,current_schemas_test';
 
-query T
+query T rowsort
 select current_schemas()::text;
 ----
-[public, current_schemas_test]
+current_schemas_test
+public
```

Comment:
There was some updating to datafusion that changes how lists are handled. I don't want this, but there's now no way for us to reference a session var that's a list and _not_ have it return a column.